apps/extension/src/routes/popup/approval/origin.tsx

```diff
@@ -36,41 +41,66 @@ export const OriginApproval = () => {
         <div className='mx-auto size-20'>
           <LinkGradientIcon />
         </div>
-        <div className='flex flex-1 flex-col items-start justify-between px-[30px] pb-[30px]'>
-          <div className='flex w-full flex-col gap-4'>
-            <div className='flex flex-col gap-2'>
-              <div className='flex h-11 w-full items-center rounded-lg border bg-background px-3 py-2 text-muted-foreground'>
-                {favIconUrl ? <img src={favIconUrl} alt='icon' className='h-11' /> : null}
-                <div className='p-2 font-headline text-base font-semibold'>
-                  {title ? title : <div className='text-muted-foreground'>{'<no title>'}</div>}
+        <div className='w-full px-[30px]'>
+          <div className='flex flex-col gap-2'>
+            <div
+              className={cn(
+                'rounded-[1em]',
+                'border-[1px]',
+                'border-transparent',
+                'p-2',
+                '[background:linear-gradient(var(--charcoal),var(--charcoal))_padding-box,_linear-gradient(to_bottom_left,rgb(139,228,217),rgb(255,144,47))_border-box]',
+              )}
+            >
```

Comment:
The favicon looks kinda unbalanced with respect to the rest of the content. Think it would be best if we resized it to fit within the normal padding of the upper box.

<img width="378" alt="Screenshot 2024-03-08 at 3 06 38 PM" src="https://github.com/penumbra-zone/web/assets/16624263/79b751f8-a483-4a3d-ac9c-e0fb83170697">

Reply:
this design is intentionally unbalanced.

the overlapping bright border is extremely visually distinct. it makes a high-contrast, unique shape. this means the origin approval popup is quickly identifiable at a glance.

the icon, likely the most recognizable and unique part of the content provided by the website, draws the eye due to its overflowing size and unusual clipped border.

i'll switch it to a round icon, it feels a bit nicer.

<img width="469" alt="Screenshot 2024-03-08 at 09 48 31" src="https://github.com/penumbra-zone/web/assets/134443988/349ed44b-86b4-4ab5-98aa-2c2f412318e7">